avocado/plugins/run.py

```diff
@@ -128,10 +128,13 @@ def configure(self, parser):
 
         if multiplexer.MULTIPLEX_CAPABLE:
             mux = parser.add_argument_group('multiplexer use on test execution')
-            mux.add_argument('-m', '--multiplex-files', nargs='*',
+            mux.add_argument('-m', '--multiplex', nargs='*', dest='multiplex_files',
                              default=None, metavar='FILE',
                              help='Location of one or more Avocado multiplex (.yaml) '
                              'FILE(s) (order dependent)')
+            mux.add_argument('--multiplex-files', nargs='*',
+                             default=None, metavar='FILE',
+                             help='DEPRECATED: please use --multiplex instead')
```

Comment:
Logging it somehow somewhere would be a nice step before fully removing it from Avocado. It could be implemented now, or a few sprints before we actually do it. It's not a big deal or a must, but I would add the warning now and then just remove the option in a few months, or years, when for whatever reason we touch this code again.

This would be my full approach:

 * Make the option an alias, keep it in --help with a deprecated message. Warning in case of usage is optional.
 * A couple of sprints later, remove the option from --help, but keep the behavior there. Log a warning if the user uses it.
 * Then a few more sprints later (2-3 more), remove the option completely, error in case it's used.

But given the priority of this particular option (not a lot of users of the multiplexer right now), I would say just killing it in a few more sprints is perfectly fine.